src/wavesurfer.js

```diff
@@ -401,7 +401,11 @@ export default class WaveSurfer extends util.Observer {
         }
 
```

Comment:
the vertical parameter needs to be documented in this file, and added to default params